src/content/content_recommender.py

```diff
@@ -38,6 +39,13 @@ def test(self, member_id, potential_events, info_repo, simscores):
         for i in xrange(len(potential_events)):
             simscores[member_id][potential_events[i]] = similarity_scores[i]
 
+        top_indices = similarity_scores.argsort()[:-5:-1]
+        top_5_recommended_events = []
+        for i in top_indices:
+            top_5_recommended_events.append(potential_events[i])
+
+        top_5_recommendation_measurement(top_5_recommended_events, info_repo["members_events"][member_id], \
```

Comment:
Shouldn't we send the (info_repo["members_events"][member_id] - training events of that members) instead of all events of that member? That is only the testing events. Accuracy should be on testing data, right? Maybe I may be thinking something wrong.